demos/TicTacToe/game_resources/proj.ios_mac/mac/main.cpp

```diff
@@ -0,0 +1,34 @@
+/****************************************************************************
```

Comment:
Unfortunatley anything that isn't copyright Google has to be in the /third_party directory of the repo.